### ISSUE TITLE
pre-ico telegram bot

### ISSUE THREAD

**Issue by tombenj** [rstormsf/ico_per_week#3]
@rstormsf what about the git repository for the pre-ico telegram bot that is on youtube? you still haven't uploaded it yet?

**Comment by rstormsf** [comment]
yeah I have it it's bunch of crap, Would you be interested to finish working on it?
